initfiles/bash/etc/init.d/install-init.in

```diff
@@ -282,22 +285,11 @@ chown root:$group ${configs}
 chown -R $user:$group ${configs}/*
 chmod 775 ${configs}
 
-# Add systemd support
-distrib_check
-if [ "${DISTRIB_NAME}" = "ubuntu" ] &&
-   [ ${DISTRIB_MAJOR_VERSION} -ge 15 ]; then
-
-   dpkg -S /sbin/init | grep -q systemd
-   if [ $? -eq 0 ]; then
-      cp ${INSTALL_DIR}/etc/init.d/hpcc-init.service /lib/systemd/system/
-      cp ${INSTALL_DIR}/etc/init.d/hpcc-init@.service /lib/systemd/system/
-      cp ${INSTALL_DIR}/etc/init.d/dafilesrv.service /lib/systemd/system/
-      cp ${INSTALL_DIR}/etc/init.d/dafilesrv@.service /lib/systemd/system/
-      systemctl daemon-reload
-   fi
-fi
+# enable systemd support if applicable
+which systemctl 1>/dev/null 2>&1 && systemctl daemon-reload
 
-createRuntime
+mkdir -p ${runtime} ${lock} ${pid} ${log}
```

Comment:
@xwang2713 Ming this should be creating that directory on installation and the following line should insure the permissions are correct.  Can you confirm that this is in the build you're using?